oss-cloud-frontend/src/axiosService.js

```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
 
-const baseURL = "http://localhost:3000";
+// const baseURL = "http://localhost:3000";
+const baseURL = "https://jhvejstuuc.execute-api.eu-west-2.amazonaws.com/aws";
```

Comment:
This can be also moved to config file. It's not necessary right now, but it will be useful in future.